### PR TITLE
Convert to luma.gl shadertools

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

File: examples/plot/plot-layer/axes-layer.js [hunk 1]
```diff
@@ -284,7 +283,7 @@ export default class AxesLayer extends Layer {
       }
     }
 
-    const labels = new Model({
+    const labels = new Model(gl, {
       gl,
```
Comment (shaojingli) [inline examples/plot/plot-layer/axes-layer.js]: remove `gl` here?

File: src/layers/core/arc-layer/arc-layer.js [hunk 2]
```diff
@@ -42,17 +42,10 @@ const defaultProps = {
...
-      shaderCache: this.context.shaderCache
-    } : {
-      vs: arcVertex,
-      fs: arcFragment,
-      modules: [],
-      shaderCache: this.context.shaderCache
-    };
+    const {shaderCache} = this.context;
+    return enable64bitSupport(this.props) ?
+      {vs: vs64, fs, modules: ['project64'], shaderCache} :
```
Comment (shaojingli) [inline src/layers/core/arc-layer/arc-layer.js]: so now fp64 modules are attached by default?

File: examples/plot/plot-layer/axes-layer.js (hunk 1 above)
Comment (ibgreen) [inline examples/plot/plot-layer/axes-layer.js]: Good catch. Fixing in separate PR.

File: src/layers/core/arc-layer/arc-layer.js (hunk 2 above)
Comment (ibgreen) [inline src/layers/core/arc-layer/arc-layer.js]: There are no default modules anymore. Instead, shader modules now can specify `dependencies` on other modules and the luma.gl `assembleShaders` automatically resolves the resulting DAG (directed acyclic graph) and injects dependencies recursively and deduplicated in the right order.

**Pull request merged**